taskcluster/tc-build-utils.sh

```diff
@@ -17,9 +17,9 @@ do_deepspeech_python_build()
 
   SETUP_FLAGS=""
   if [ "${package_option}" = "--cuda" ]; then
-    SETUP_FLAGS="--project_name deepspeech-gpu"
+    SETUP_FLAGS="--project_name mozilla_voice_stt-gpu"
```

Comment:
so we mix? not a big fan :)
also, maybe we should name `-gpu` as `-cuda`

Reply:
I actually like it a bit to separate the project name from the package type modifier. But I can change to `_gpu`/`_cuda` if you prefer.

Reply:
well, the thing is we won't be able to ship cuda + rocm or + opencl under the same package, so it's better we take the opportunity we have to rename to ease our future

Reply:
Going with underline, and `_cuda` then.